bot/exts/events/advent_of_code/_cog.py

```diff
@@ -185,15 +186,47 @@ async def join_leaderboard(self, ctx: commands.Context) -> None:
         brief="Get a snapshot of the PyDis private AoC leaderboard",
     )
     @whitelist_override(channels=AOC_WHITELIST_RESTRICTED)
-    async def aoc_leaderboard(self, ctx: commands.Context) -> None:
-        """Get the current top scorers of the Python Discord Leaderboard."""
+    async def aoc_leaderboard(
+            self,
+            ctx: commands.Context,
+            dayandstar: Optional[str] = None,
+            maximum_scorers: Optional[int] = 10) -> None:
```

Comment:
```suggestion
            maximum_scorers: Optional[int] = 10,
        ) -> None:
```
Not sure if I am mis-remembering the code style, but should the closing of the function be on its own line?

Reply:
Yes, It would make sense I believe. Thanks for pointing out, I totally missed that one. 👍